client/src/curriculum/modules-list.scss

```diff
@@ -0,0 +1,236 @@
+@use "../ui/vars" as *;
+
+.curriculum-content-container.with-sidebar,
+.curriculum-content-container {
+  .curriculum-content {
+    .modules {
+      input[type="radio"]:not(:checked) ~ a,
+      input[type="radio"]:not(:checked) ~ ol {
+        display: none;
+      }
+    }
+
+    .module-contents {
+      h2 {
+        width: 100%;
+      }
+    }
+
+    ol.modules-list-list {
+      display: grid;
+      grid-template-areas:
+        "a   b   c"
+        "hr  hr  hr"
+        "mod mod mod"
+        "t   t   t";
+
+      grid-template-columns: auto;
+      margin: 0;
+      padding: 0;
+      @media screen and (min-width: $screen-sm) {
+        grid-template-areas:
+          "a   b   c   spacer"
+          "hr  hr  hr  hr"
+          "mod mod mod mod"
+          "t   t   t   t";
+
+        grid-template-columns: auto auto auto 1fr;
+      }
+
+      hr {
+        border: none;
+        border-top: 1px solid var(--text-inactive);
+        grid-area: hr;
+        margin: 0 0 1.5rem;
+        width: 100%;
+      }
+
+      li {
```

Comment:
It would be helpful to have a comment explaining what this list represents, or assign it a labeling CSS class for that matter. (I guess it's the Getting started / Core / Extensions tabs?)